src/parameterizations/vertical/MOM_diabatic_driver.F90

```diff
@@ -3558,6 +3565,23 @@ subroutine diabatic_driver_init(Time, G, GV, US, param_file, useALEalgorithm, di
 
 end subroutine diabatic_driver_init
 
+!> Routine to register restarts, pass-through to children modules
+subroutine register_diabatic_restarts(G, US, param_file, int_tide_CSp, restart_CSp)
+  type(ocean_grid_type), intent(in)    :: G           !< The ocean's grid structure
+  type(unit_scale_type), intent(in)    :: US          !< A dimensional unit scaling type
+  type(param_file_type), intent(in)    :: param_file  !< A structure to parse for run-time parameters
+  type(int_tide_CS),     pointer       :: int_tide_CSp !< Internal tide control structure
+  type(MOM_restart_CS),  pointer       :: restart_CSp  !< MOM restart control structure
+
+  logical :: use_int_tides = .false.
```

Comment:
This variable should be initialized outside of declaration.  (In fact, I don't think there's any need to initialize it at all.)